tests/test_pyiges.py

```diff
@@ -114,6 +114,28 @@ def test_surfaces_vtk(surf):
                                          -9.363301218200274, -45.772995131000016, -8.876323512))
 
 
+def test_surfaces_geomdl(surf):
+    gsurf = surf.to_geomdl()
+    assert gsurf.data['control_points'] == pytest.approx((
+        [-26.902905334, -16.51153913, -8.876323512],
+        [-25.851820351, -15.866440371, -21.167794775],
+        [-25.995725563, -15.954761558, -33.519826531],
+        [-27.332763631, -16.775362758, -45.772995131],
+        [-28.232974775, -14.344404257, -8.876323512],
+        [-27.129924549, -13.783974529, -21.167794775],
+        [-27.28094438, -13.860703585, -33.519826531],
+        [-28.684085103, -14.573601108, -45.772995131],
+        [-29.292803148, -12.033057876, -8.876323512],
+        [-28.148345882, -11.562931461, -21.167794775],
+        [-28.305034796, -11.62729699, -33.519826531],
+        [-29.760847559, -12.225323719, -45.772995131],
+        [-30.067010387, -9.611041892, -8.876323512],
+        [-28.892305175, -9.235542604, -21.167794775],
+        [-29.05313537, -9.286952628, -33.519826531],
+        [-30.547425187, -9.764608432, -45.772995131],
+    ))
```

Comment:
I also added some tests to increase coverage for the `to_geomdl` functionality which is relevant for this topic.

When *not* installing as [full], all tests related to pyvista/vtk give the desired expection message:

![grafik](https://github.com/pyvista/pyiges/assets/17496175/e1ce2b41-80fa-48cf-a247-f3267db15725)

Reply:
Note that the tests are not written depending on `_IS_FULL_MODULE`, i.e. they only work for the full module.

It would be more precise, but also more effort, to adapt them such that they are decorated with a `pytest.raises` for the pure parsing variant. However, this would only make sense IMO, if the workflow would also be extended to *two pytest runs* for `pyiges[full]` and `pyiges`. Please let me know if You would like this improvement to be added.